README.md

```diff
@@ -142,7 +142,7 @@ See [Kore.io] documentation for more details on various ways to run the server.
 For example, the following commmand will run the `backend` server as a foreground process in DEBUG mode all log output will be visible in the terminal console:
 
 ```
-CB_HOST=127.0.0.1 CB_USER=raycardillo CB_PSWD=raycardillo ./dev-run.sh
+CB_HOST=127.0.0.1 CB_USER=raycardillo CB_PSWD=raycardillo ./run-dev.sh
```

Comment:
Not for this line, but for the README in general, since Docker works now (🥳 ) we can get rid of the warnings. For example:
https://github.com/couchbaselabs/try-cb-lcb/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R59